src/grammar.json

```diff
@@ -581,7 +653,9 @@
       "storm",
       "still",
       "tron",
-      "erator"
+      "erator",
+      "-troll",
```

Comment:
“I just threw it together this morning……I wasn’t even planning on playing this event…..I was in the neighborhood and figured why not……” Then he made a face like this:
:-9